tools/block-generator/generator/generate.go

```diff
@@ -80,6 +104,7 @@ func MakeGenerator(dbround uint64, bkGenesis bookkeeping.Genesis, config Generat
 		rewardsRate:               0,
 		rewardsRecalculationRound: 0,
 		reportData:                make(map[TxTypeID]TxData),
+		latestData:                make(map[TxTypeID]uint64),
```

Comment:
Helpful for debugging, this keeps a count of how many transactions of each **txType** occured in the current round.